index.html

```diff
@@ -5,11 +5,164 @@
+                <h2>Main Program</h2>
+                <div class="line"></div>
+            </div>
+            <div class="m-program">
+                <div class="h-main" id="main-1">
+                    <h2>Main Program</h2>
+                    <div class="line"></div>
+                </div>
+                <ul class="card card-1" id="card-a">
+                    <li class="item a"><img class="speaker" src="images/speaker.png" alt="speaker"></li>
+                    <li class="item b">Orientation</li>
+                    <li class="item c">
+                        <p>Listen to the teamleader from various countries make an opening presentation</p>
+                    </li>
+                </ul>
+                <ul class="card card-1" id="card-b">
+                    <li class="item a"><img class="speaker" src="images/speaker.png" alt="speaker"></li>
+                    <li class="item b">Orientation</li>
+                    <li class="item c">
+                        <p>Listen to the teamleader from various countries make an opening presentation</p>
+                    </li>
+                </ul>
+                <ul class="card card-1" id="card-c">
+                    <li class="item a"><img class="speaker" src="images/speaker.png" alt="speaker"></li>
+                    <li class="item b">Orientation</li>
+                    <li class="item c">
+                        <p>Listen to the teamleader from various countries make an opening presentation</p>
+                    </li>
+                </ul>
+                <ul class="card card-1" id="card-d">
+                    <li class="item a"><img class="speaker" src="images/speaker.png" alt="speaker"></li>
+                    <li class="item b">Orientation</li>
+                    <li class="item c">
+                        <p>Listen to the teamleader from various countries make an opening presentation</p>
+                    </li>
+                </ul>
+                <ul class="card card-1" id="card-e">
+                    <li class="item a"><img class="speaker" src="images/speaker.png" alt="speaker"></li>
+                    <li class="item b">Orientation</li>
+                    <li class="item c">
+                        <p>Listen to the teamleader from various countries make an opening presentation</p>
+                    </li>
+                </ul>
+            </div>
+
+        </section>
+        <section class="featured-speakers">
+            <div class="h-featured">
+                <h2>Featured speakers</h2>
+                <div class="line"></div>
+            </div>
+            <div class="speaker-grid" id="speaker-less">
+            </div>
+            <div class="speaker-grid" id="speaker-all"></div>
+            <button class="btn-mobile-only show-more" id="show-more" type="button">Show more</button>
+            <button class="btn-mobile-only show-less" id="show-less" type="button">Show less</button>
+
+        </section>
```

Comment:
- [ ] On the feature speaker's section, the images are too stretched. Try and reduce the height of the images.

Reply:
Hey emmuellekamwa, that change requested by the previous CR, been mate for the home page it not true it is white not dark but for the about page it is dark 
you can also confirm that yourself

Reply:
Sorry to say so but it has not been met. If I am not mistaken your working branch is called dynamic-js. When I click on the live demo I can't see that the images issue has been met and also for the background color of the footer on desktop view the background colors on both pages are dark and on mobile view it is white. I say so because it is part of the requirements on my reviewer's guide